github_pr.py

```diff
@@ -179,6 +240,14 @@ def main():
     github-pr list -r dataxu/test_repo
     github-pr list -r dataxu/test_repo -n 17
 
+    - Filters - can be used alone or together
+        --filters
+            * owner - This will return a list of PRs from the repo that are owned by the github-user
+            * status - returns PRs with a specifc status, one of (success, failure, error, pending)
+            * label - returns PRs with a specifc label
+            * comments - returns PRs whos comments contain a string
+      github-pr list -r dataxu/test_repo --filters 'filter1_name=filter1_value,filter2_name=filter2_value'
```

Comment:
can you add some examples of calling with different combined or single filters, e.g. `github-pr list -r dataxu/test_repo --filters 'owner=okely-dokely,status=success'`?